test/server/cards/02_SHD/units/GentleGiant.spec.ts

```diff
@@ -0,0 +1,60 @@
+describe('Gentle Giant', function() {
+    integration(function(contextRef) {
+        describe('Gentle Giant\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [
+                            { card: 'gentle-giant', damage: 1 },
+                            { card: 'r2d2#ignoring-protocol', damage: 3 },
+                            { card: 'c3po#protocol-droid', damage: 1 }],
+                    },
+                    player2: {
+                        groundArena: [{ card: 'wampa', damage: 1 }]
+                    }
+                });
+            });
+
+            it('heal a target', function () {
+                const { context } = contextRef;
+
+                const reset = (passAction = true) => {
+                    context.gentleGiant.exhausted = false;
+                    context.r2d2.damage = 3;
+                    if (passAction) {
+                        context.player2.passAction();
+                    }
+                };
+
+                // CASE 1: Heal a target
+                // Attack
+                context.player1.clickCard(context.gentleGiant);
+                expect(context.gentleGiant).toBeInLocation('ground arena');
+                expect(context.player1).toBeAbleToSelectExactly([context.p2Base, context.wampa]);
```

Comment:
Don't need these checks since they're not really testing the card implementation. We can safely assume that test setup and basic stuff like attack just works since we have other tests that cover them, want to keep card tests as streamlined as possible